conf/pbspro.config

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
Hey @bentsherman. I don't think these type of scheduler specific profiles should be in an nf-core workflow.  User's are meant to create their own local `nextflow.config` files with these types of settings inside of them specific to their own infrastructure.  The config files that come with an nf-core workflow, as per my understanding, must be generic.